shell/tests/apply_and_verify_context_actions_from_file.rs

```diff
@@ -0,0 +1,172 @@
+    let schemas = vec![
+        storage::block_storage::BlockPrimaryIndex::descriptor(&cache),
+        storage::block_storage::BlockByLevelIndex::descriptor(&cache),
+        storage::block_storage::BlockByContextHashIndex::descriptor(&cache),
+        storage::BlockMetaStorage::descriptor(&cache),
+        storage::OperationsStorage::descriptor(&cache),
+        storage::OperationsMetaStorage::descriptor(&cache),
+        storage::context_action_storage::ContextActionByBlockHashIndex::descriptor(&cache),
+        storage::context_action_storage::ContextActionByContractIndex::descriptor(&cache),
+        storage::context_action_storage::ContextActionByTypeIndex::descriptor(&cache),
+        storage::ContextActionStorage::descriptor(&cache),
+        storage::merkle_storage::MerkleStorage::descriptor(&cache),
+        storage::SystemStorage::descriptor(&cache),
+        storage::persistent::sequence::Sequences::descriptor(&cache),
+        storage::MempoolStorage::descriptor(&cache),
+        storage::ChainMetaStorage::descriptor(&cache),
+        storage::PredecessorStorage::descriptor(&cache),
+    ];
+
+    let db_config = storage::persistent::DbConfiguration::default();
+    storage::persistent::open_kv(path, schemas, &db_config)
+        .map(Arc::new)
+        .unwrap()
+}
+
+fn get_action_symbol(action: &ContextAction) -> String {
+    match action{
+        ContextAction::Set {..} => {String::from("Set")},
+        ContextAction::Delete {..} => {String::from("Delete")},
+        ContextAction::RemoveRecursively {..} => {String::from("RemoveRecursively")},
+        ContextAction::Copy {..} => {String::from("Copy")},
+        ContextAction::Checkout {..} => {String::from("Checkout")},
+        ContextAction::Commit {..} => {String::from("Commit")},
+        ContextAction::Mem {..} => {String::from("Mem")},
+        ContextAction::DirMem {..} => {String::from("DirMem")},
+        ContextAction::Get {..} => {String::from("Get")},
+        ContextAction::Fold {..} => {String::from("Fold")},
+        ContextAction::Shutdown {..} => {String::from("Shutdown")},
+    }
+}
+
+#[test]
+fn feed_tezedge_context_with_actions() -> Result<(), Error> {
+
+    let block_header_stub = BlockHeaderBuilder::default()
+        .level(0)
+        .proto(0)
+        .predecessor(vec![0;32])
+        .timestamp(0)
+        .validation_pass(0)
+        .operations_hash(vec![0;32])
+        .fitness(vec![])
+        .context(vec![0;32])
+        .protocol_data(vec![])
+        .build()
+        .unwrap();
+
+    let header_stub = BlockHeaderWithHash::new(block_header_stub).unwrap();
+
+    let cache = Cache::new_lru_cache(128 * 1024 * 1024).unwrap(); // 128 MB
+    let commit_log_db_path = PathBuf::from("/tmp/commit_log/");
+    let key_value_db_path = PathBuf::from("/tmp/key_value_store/");
+    // let actions_storage_path = PathBuf::from("/mnt/hdd/milion.bin");
+    let actions_storage_path = PathBuf::from("/mnt/hdd/node-data/actionfile.bin");
```

Comment:
This will not work on other people's computers. @bkontur what should be used here? `OUT_DIR` ? (I think that is what is used in other tests)

Reply:
Oh wait, just noticed that the file is read, not written to, and also not generated by the build process or tests. Can it be a path related to the repo directory? something not absolute (and if not there, it should be reported with an error). Not sure how this is going to be handled on CI.

Reply:
Right i forgot to disable it - the idea is to create network storage where this file will be accessible on CI - but not as a part of this PR ;)